src/ViewModel/SongLyricViewModel.h

```diff
@@ -28,6 +28,8 @@ class SongLyricViewModel : public QAbstractListModel {
 
     QHash<int, QByteArray> roleNames() const override;
 
+    QList<SongLyric> ProcessLyric (QString);
```

Comment:
```suggestion
    QList<SongLyric> ProcessLyric(QString);
```

Follow our code style defined in `clang-format.yaml`, if there is no specific reason.

Reply:
use **pass by const reference** for large object :eyes: